go.mod

```diff
@@ -10,14 +10,14 @@ require (
 	github.com/open-policy-agent/cert-controller v0.3.0
 	github.com/prometheus/client_golang v1.12.1
 	go.uber.org/zap v1.21.0
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
-	k8s.io/component-base v0.23.3
-	k8s.io/component-helpers v0.23.4
-	k8s.io/klog/v2 v2.40.1
+	k8s.io/api v0.24.0
+	k8s.io/apimachinery v0.24.0
+	k8s.io/client-go v0.24.0
+	k8s.io/component-base v0.24.0
+	k8s.io/component-helpers v0.24.0
+	k8s.io/klog/v2 v2.60.1
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
-	sigs.k8s.io/controller-runtime v0.11.1
+	sigs.k8s.io/controller-runtime v0.12.1-0.20220614131733-a759a0d51676
```

Comment:
same as above

Reply:
0.12.1 has not yet included the fixed pr.

Reply:
oh, then let's not merge this until there's a release

/approve cancel